bool is_active
---
string result